InvenTree/part/test_part.py

```diff
@@ -631,7 +631,7 @@ def _notification_run(self, run_class=None):
         self.part.save()
 
         # There should be 1 notification
-        self.assertEqual(NotificationEntry.objects.all().count(), 1)
+        self.assertEqual(NotificationEntry.objects.all().count(), 2)
```

Comment:
Question: Why should be this be higher?

Reply:
I believe it is because as part of the notification unit testing there's a broken notification, which logs an error, which in turn sends another notification.

Reply:
The distinction in NotImplemented and Exception was so that Test ng etc. is kept somewhat predictable.